### ISSUE TITLE
Include Biography in Artist's Metadata

### ISSUE THREAD

**Issue by zsaraf** [spotify/web-api#207]
The EchoNest API had the artist biography endpoint which was incredibly helpful, but the Spotify Web API does not support this. I think this was a great feature, and Spotify obviously has this data as it has a bio for most somewhat famous artists. Would be really helpful if it was added to the Spotify Web API!

Thanks in advance and hope that Spotify can add this endpoint soon!

**Comment by jch254** [comment]
Yes please! Location information of artists would be ideal

**Comment by eugenio11** [comment]
I agree, the echonest bio method was very useful!

**Comment by sheldonv** [comment]
Is there another api that offers this service?

**Comment by zsaraf** [comment]
after doing a lot of research, we found that last.fm actually has great crowdsourced artist bio's. Check out their selection by visiting their developer website and trying out a few artists

**Comment by sheldonv** [comment]
Thanks, very helpful, also do you know of APIs that offer artist blogs and recent reviews.

**Comment by alexpersian** [comment]
So is this endpoint still reliant on the EchoNest API? I'm assuming Spotify's main app is using that information for the bio's there.

**Comment by simskij** [comment]
So, any news on this? I think it'd be very helpful.

And to elaborate on @sheldonvs post, if anyone has any suggestion on alternative APIs to use for atleast semi-reliable artist biographies in the mean time I'd appreciate it kindly. Especially if it's usable without auth, much like the Spotify WebApi.

**Comment by alexpersian** [comment]
@simskij what I ended up doing was using the last.fm API to pull in artist bios.

**Comment by simskij** [comment]
Thanks! That was my thought as well, but Spotify really should fix this to prevent loosing api consumers to last.fm.






On Sat, Jul 30, 2016 at 7:40 PM +0200, "Alex Persian" <notifications@github.com> wrote:





@simskij what I ended up doing was using the last.fm API to pull in artist bios.

---

**Comment by sheldonv** [comment]
I've incorporated that endpoint also, but Echonest offered Blogs and
Reviews which were very helpful.

**Comment by Gdewilde** [comment]
@thelinmichael do you have any information on this? I'm looking into last.fm as an alternative atm.. Thanks.

**Comment by icebox** [comment]
What up, Spotify?  This request has been open for almost two years. You have the data. It existed in Echo Nest. You make so much money and pay artist next to nothing. The least you can do is offer them as much exposure as possible, no?

**Comment by sajsanghvi** [comment]
@icebox Agreed lol but they also lose hella money every quarter

**Comment by mjw56** [comment]
I'm a building a small spotify mini-clone as a learning project. It would be sweet to have access to this info from the API!

Maybe the same data this is at `https://spclient.wg.spotify.com/open-backend-2/v1/artists/5oOhM2DFWab8XhSdQiITry` on the `artist` endpoint? I tried to query this but get a `403`. 🙂

**Comment by arirawr** [comment]
Hey all,

As per our [new developer support process](https://github.com/spotify/web-api#creating-an-issue), any feature requests that have no plans to be considered in the next 6 months will be closed as wontfix.

We do not have any plans to implement this in the next 6 months and thus will close as wontfix.

We encourage you and other members of the community to continue to share use cases and your support for this feature, as it helps us prioritize features going forward. You will be able to comment on this issue even after it is closed. We will also post any updates here relating to this suggestion when they happen.

**Comment by Havock94** [comment]
Still missing biography in the APIs, after almost an year later, is there any plan on providing an artist's biography?

**Comment by arirawr** [comment]
Hi @Havock94, we do not currently have any plans to implement this. We'll update here if that changes.

**Comment by scgough** [comment]
Hi @arirawr 
I know you've only recently posted but I'd like to also place my request for this as well. 
I think the artist biography would be a very useful addition to the API. 

From the data you provide if I wanted to create an artist page on my site at the moment it would be very sparse. 
I would most likely use the name, link to the artist on Spotify and an image. I would love to be able to add the biog to that.  

My only option from a development POV would be to try and retrieve the biography from a 3rd party service (_of which there doesn't seem to be a decent option for out there_) or to scrape it from the web somehow (_which I won't and don't want to do_). 

In addition, if I did find a good enough 3rd party solution the match of artist=>artist across the 2 services would be fuzzy at best and rely of exact artist name matches. 

There used to be several APIs out there offering artist meta data but many have seemingly died out over the years. I believe that adding this additional metadata to your API would actually on-board more developers over time. 

To sum up, there is definitely a gap that needs filling here. Please help.
